sources/packages/web/src/services/http/InstitutionUserApi.ts

```diff
@@ -0,0 +1,151 @@
+import HttpBaseClient from "./common/HttpBaseClient";
+import { PaginatedResults, ClientIdType, PaginationOptions } from "@/types";
+import {
+  InstitutionUserAPIOutDTO,
+  InstitutionUserDetailAPIOutDTO,
+  InstitutionUserStatusAPIOutDTO,
+  CreateInstitutionUserAPIInDTO,
+  UpdateInstitutionUserAPIInDTO,
+  UserActiveStatusAPIInDTO,
+} from "@/services/http/dto";
+import { AuthService } from "../AuthService";
+import { getPaginationQueryString } from "@/helpers";
+
+export class InstitutionUserApi extends HttpBaseClient {
+  /**
+   * Get details of user who is logged in.
+   * @returns user details.
+   */
+  async getMyInstitutionDetails(
+    header?: any,
```

Comment:
param comment

Reply:
I am removing the header from these methods. This is a leftover. I was not able to find a reason to keep them.

Reply:
Changed!